eval.py

```diff
@@ -22,32 +22,35 @@
     print file        
     startTime = datetime.now()
 
-    c = ExcelCompiler(file, ignore_sheets = ['IHS'])
-    c.clean_volatile()
-    print "___Timing___ %s cells and %s named_ranges parsed in %s" % (str(len(c.cells)-len(c.named_ranges)), str(len(c.named_ranges)), str(datetime.now() - startTime))
-    sp = c.gen_graph(outputs=["outNPV_Proj"])
-    print "___Timing___ Graph generated in %s" % (str(datetime.now() - startTime))
+    # c = ExcelCompiler(file, ignore_sheets = ['IHS'])
+    # c.clean_volatile()
+    # print "___Timing___ %s cells and %s named_ranges parsed in %s" % (str(len(c.cells)-len(c.named_ranges)), str(len(c.named_ranges)), str(datetime.now() - startTime))
+    # sp = c.gen_graph(outputs=["outNPV_Proj"])
+    # print "___Timing___ Graph generated in %s" % (str(datetime.now() - startTime))
     
     # print "Serializing to disk...", file
     # sp.dump(file.replace("xlsx", "gzip"))
 
 
-    # startTime = datetime.now()
-    # print "Reading from disk...", file
-    # sp = Spreadsheet.load(file.replace("xlsx", "gzip"))
-    # print "___Timing___ Graph read in %s" % (str(datetime.now() - startTime))
+    startTime = datetime.now()
+    print "Reading from disk...", file
+    sp = Spreadsheet.load(file.replace("xlsx", "gzip"))
```

Comment:
Can we manage inputs and ouputs in main() signature in order to clean koala from specific variable names ?